build.sbt

```diff
@@ -6,7 +6,7 @@ organization in Global := "edu.gemini.ocs"
 
 ocsVersion in ThisBuild := OcsVersion("2016A", true, 1, 1, 1)
 
-pitVersion in ThisBuild := OcsVersion("2018A", false, 1, 1, 0)
+pitVersion in ThisBuild := OcsVersion("2018A", false, 1, 1, 1)
```

Comment:
Why are we adding another point version to the PIT? (Not that I think it's a bad idea... just curious.)

Reply:
Because i'm releasing a minor fix for p1-monitor who uses the same version as the pit

Reply:
Oh, of course. Sorry for the stupid question. 😳